app/dose-notifications/dose-notifications.ts

```diff
@@ -0,0 +1,34 @@
+import {Component} from "@angular/core";
+import {NavController, Alert} from 'ionic-angular';
+import {LocalNotifications} from 'ionic-native';
+
+export class DoseNotifications {
+
+    constructor(private navController: NavController) {
+        LocalNotifications.on("click", (notification, state) => {
+            let alert = Alert.create({
+                title: "Dose Amigos Medication Notification",
+                subTitle: "You currently have pending medications.",
+                buttons: ["OK"]
+            });
+            this.navController.present(alert);
+        });
+    }
```

Comment:
What does this constructor do?

Presumably this is meant to be a service and we would add `DoseNotifications` to a provide in app.ts?
At which time it would be constructed, so it would show this alert when the app starts?
Please explain.